ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -193,15 +193,11 @@ objects:
         trap 'rc=$?; if [[ $rc -ne 0 ]]; then touch /tmp/shared/exit; else touch /tmp/shared/install; fi; exit $rc' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
-        # Don't expand IMAGE_FORMAT here - it contains '${component}'
-        oreg_url='${IMAGE_FORMAT}'
-        if [[ -n "${PREVIOUS_IMAGE_TAG}" ]]; then
-          oreg_url='docker.io/openshift/origin-${component}:${PREVIOUS_IMAGE_TAG}'
-        fi
-
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
-          -e oreg_url="${oreg_url}" \
+          -e oreg_url='docker.io/openshift/origin-${component}:v${PREVIOUS_ANSIBLE_VERSION}' \
+          -e openshift_release="${PREVIOUS_ANSIBLE_VERSION}" \
+          -e openshift_version="${PREVIOUS_ANSIBLE_VERSION}.0" \
```

Comment:
Why set openshift_version?  That's not something we should be setting and if we are, something else is broken.

Reply:
healthchecks would fail when only `openshift_release` is set - it would complain newer packages are available